internal/backend/local/backend_plan_test.go

```diff
@@ -161,6 +161,39 @@ func TestLocal_plan_context_error(t *testing.T) {
 	}
 }
 
+// Ensure that the plantimestamp() call is not affecting the validation step during planning.
+func TestLocal_plan_range_over_zero_plan_timestamp(t *testing.T) {
```

Comment:
This test should probably be a TestContext2Validate test instead, in context_validate test.